<launch>
  <arg name="input/pose_with_cov" default="/localization/pose_twist_fusion_filter/pose_with_covariance" />

  <node name="localization_error_monitor" exec="localization_error_monitor" pkg="localization_error_monitor" output="screen">
    <remap from="input/pose_with_cov" to="$(var input/pose_with_cov)" />
    <param name="scale" value="3.0" />
    <param name="error_ellipse_size" value="1.0" />
    <param name="warn_ellipse_size" value="0.8" />
  </node>
</launch>
